top.sls

```diff
@@ -1,11 +1,14 @@
 base:
```

Comment:
I think we want to create a category just for obs

just create a fake category of machines
'E@obs-blade*':